src/components/Departments/DepartmentDetailComponent.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { isEmpty } from 'lodash';
```

Comment:
lodash is quite a big library and will make the overall bundle heavy. It'd be great to consider rambda, a lightweight alternative